securedrop_salt/sd-app.sls

```diff
@@ -55,15 +55,3 @@ sd-app-private-volume-size:
         qvm-volume resize sd-app:private {{ d.vmsizes.sd_app }}GiB
     - require:
       - qvm: sd-app
-
-# Ensure the Qubes menu is populated with relevant app entries,
```

Comment:
I went back and forth on whether removing this was a good idea or not, maybe we shouldn't be making unnecessary changes now that we're in the RC testing phase?